<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgesWidget</class>
 <widget class="QWidget" name="EdgesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btHighlightSelectedEdges">
       <property name="toolTip">
        <string>Highlight in network views the selected edges in the table</string>
       </property>
       <property name="statusTip">
        <string>Highlight in network views the selected edges in the table</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../ui.qrc">
         <normaloff>:/icons/images/highlight-red.svg</normaloff>:/icons/images/highlight-red.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btHighlightNodesFromSelectedEdges">
       <property name="toolTip">
        <string>Highlight in network views the nodes connected by selected edges in the table</string>
       </property>
       <property name="statusTip">
        <string>Highlight in network views the nodes connected by selected edges in the table</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../ui.qrc">
         <normaloff>:/icons/images/highlight-yellow.svg</normaloff>:/icons/images/highlight-yellow.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btSetAlternatingRowColors">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../ui.qrc">
         <normaloff>:/icons/images/alternating_row_colors.svg</normaloff>:/icons/images/alternating_row_colors.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btEnableOrdering">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../ui.qrc">
         <normaloff>:/icons/images/enable_ordering.svg</normaloff>:/icons/images/enable_ordering.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btSetColumnsMovable">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../ui.qrc">
         <normaloff>:/icons/images/enable_move_columns.svg</normaloff>:/icons/images/enable_move_columns.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="EdgeTableView" name="tvEdges">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionHighlightSelectedEdges">
   <property name="icon">
    <iconset resource="../../ui.qrc">
     <normaloff>:/icons/images/highlight-red.svg</normaloff>:/icons/images/highlight-red.svg</iconset>
   </property>
   <property name="text">
    <string>Highlight Selected Edges</string>
   </property>
   <property name="toolTip">
    <string>Highlight in network views the selected edges in the table</string>
   </property>
   <property name="statusTip">
    <string>Highlight in network views the selected edges in the table</string>
   </property>
  </action>
  <action name="actionHighlightNodesFromSelectedEdges">
   <property name="icon">
    <iconset resource="../../ui.qrc">
     <normaloff>:/icons/images/highlight-yellow.svg</normaloff>:/icons/images/highlight-yellow.svg</iconset>
   </property>
   <property name="text">
    <string>Highlight nodes from selected edges</string>
   </property>
   <property name="toolTip">
    <string>Highlight in network views the nodes connected by selected edges in the table</string>
   </property>
   <property name="statusTip">
    <string>Highlight in network views the nodes connected by selected edges in the table</string>
   </property>
  </action>
  <action name="actionSetAlternatingRowColors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../ui.qrc">
     <normaloff>:/icons/images/alternating_row_colors.svg</normaloff>:/icons/images/alternating_row_colors.svg</iconset>
   </property>
   <property name="text">
    <string>Set Alternating Row Colors</string>
   </property>
   <property name="toolTip">
    <string>Alternate background colors in table to improve readability</string>
   </property>
   <property name="statusTip">
    <string>Alternate background colors in table to improve readability</string>
   </property>
  </action>
  <action name="actionEnableOrdering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../ui.qrc">
     <normaloff>:/icons/images/enable_ordering.svg</normaloff>:/icons/images/enable_ordering.svg</iconset>
   </property>
   <property name="text">
    <string>Enable Ordering</string>
   </property>
   <property name="toolTip">
    <string>Enable ordering of table. Selection of columns is available only when sorting is disabled</string>
   </property>
   <property name="statusTip">
    <string>Enable ordering of table. Selection of columns is available only when sorting is disabled</string>
   </property>
  </action>
  <action name="actionSetColumnsMovable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../ui.qrc">
     <normaloff>:/icons/images/enable_move_columns.svg</normaloff>:/icons/images/enable_move_columns.svg</iconset>
   </property>
   <property name="text">
    <string>Set Columns Movable</string>
   </property>
   <property name="toolTip">
    <string>Allow to change order of columns</string>
   </property>
   <property name="statusTip">
    <string>Allow to change order of columns</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EdgeTableView</class>
   <extends>QTableView</extends>
   <header>metgem_app/ui/widgets/metadata/edge.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ui.qrc"/>
 </resources>
 <connections/>
</ui>
